staging/src/k8s.io/api/Gopkg.toml

```diff
@@ -0,0 +1,10 @@
+[[constraint]]
```

Comment:
the Godeps.json files in the staging dirs have not changed? Do we still generated them?

Reply:
We have to because the publishing-bot depends on them, and people vendoring with godep or glide depend on them.

Reply:
> But it seems you're saying we're also going to need valid Godeps.json files no matter what.

I think until vgo is official and adopted wide enough, we will have to keep Godeps.json around, largest common denominator. It's sad, yes.

Potentially we can simulate a `godep restore` run based on `gopkg.lock` in the publisher bot, i.e. a run which populates the GOPATH.

Reply:
I tried a few things, and settled for a feature in `dep` to generate all those files after a solution is found (at this point we have an exact knowledge of what packages are needed at what version for what `k8s.io/*` repo). I guess that would mean we could retire `hack/update-staging-godeps-dockerized.sh` ?

Incidentally it seems the currently generated `Godeps.json` can have duplicated entries, which is slightly weird.

Reply:
> duplicated entries

You mean multiple packages per repo? That's normal for Godep. And the bot makes use of that to avoid running godep safe+restore when nothing changed. So I fear we need that as well.

Reply:
no I mean multiple entries for the same package. See the diff for `staging/src/k8s.io/api/Godeps/Godeps.json` for example. The removed entries are the ones that were previously duplicated.

Reply:
That's indeed weird.

Reply:
The generated Godeps.json looks fine now. The publishing bot should work with that.